src/integrations/contact-input.tsx

```diff
@@ -139,6 +142,7 @@ type ContactInput = {
 	defaultValue: Array<Contact>;
 	placeholder: string;
 	background?: keyof DefaultTheme['palette'];
+	chipDisplayName?: string;
```

Comment:
string is too generic and we loose the suggestions of the ide. Use instead something like 
```suggestion
	chipDisplayName?: typeof CHIP_DISPLAY_NAME_VALUE[keyof typeof CHIP_DISPLAY_NAME_VALUE];
```